src/main/java/build/buildfarm/BUILD

```diff
@@ -74,6 +74,77 @@ java_library(
     ],
 )
 
+java_library(
+    name = "common-metrics",
+    srcs = glob([
+        "common/metrics/AbstractMetricsPublisher.java",
+        "common/metrics/MetricsPublisher.java",
+    ]),
+    deps = [
+        "@maven//:com_google_guava_guava",
+        "@maven//:com_google_protobuf_protobuf_java",
+        "@maven//:com_google_protobuf_protobuf_java_util",
+        "//src/main/protobuf:build_buildfarm_v1test_buildfarm_java_proto",
```

Comment:
deps ordering here and below.

Reply:
We should look at the rest of the build files as well as the same ordering exists in other places.